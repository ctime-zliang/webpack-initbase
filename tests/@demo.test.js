/*
    当前文件中所有测试执行前触发, 只执行一次
 */
// beforeAll(() => {
// 	console.log('@demo.test - beforeAll')
// })
/*
    当前文件中每个测试执行前都会触发
 */
// beforeEach(() => {
// 	console.log('@demo.test - beforeEach')
// })
/*
    当前文件中每个测试结束后都会触发
 */
// afterEach(() => {
// 	console.log('@demo.test - afterEach')
// })
/*
    当前文件中所有测试执行结束后触发, 只执行一次
 */
// afterAll(() => {
// 	console.log('@demo.test - afterAll')
// })

describe(`JavaScript Test Demo`, () => {
	test(`Match Snapshot Demo`, () => {
		const user = {
			id: '234',
			name: 'LeBron James',
		}
		expect(user).toMatchSnapshot()
	})
})
