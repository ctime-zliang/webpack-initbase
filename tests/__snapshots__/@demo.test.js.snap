// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`JavaScript Test Demo Match Snapshot Demo 1`] = `
Object {
  "id": "234",
  "name": "LeBron James",
}
`;
